src/client/app/containers/LineChartContainer.ts

```diff
@@ -31,19 +24,50 @@ function mapStateToProps(state: State) {
 					throw new Error('Unacceptable condition: readingsData.readings is undefined.');
 				}
 
-				const dataPoints: ChartPoint[] = _.values(readingsData.readings).map(compressedReadingToChartPoint);
+				// Create two arrays for the x and y values. Fill the array with the data from the compressed readings
+				const xData: string[] = [];
+				const yData: number[] = [];
+				const hoverText: string[] = [];
+				const readings = _.values(readingsData.readings);
+				readings.forEach(reading => {
+					const timeReading = moment(reading.startTimestamp);
```

Comment:
The code here differs slighly from BarCharContainer.ts. They do the same basic work so it would be nice if they did it in a similar way.

Reply:
Addressed in commit 62e075f